handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1076,21 +1076,31 @@ public void setEnabledProtocols(String[] protocols) {
                 // Enable all and then disable what we not want
                 SSL.setOptions(ssl, SSL.SSL_OP_ALL);
 
+                int opts = SSL.SSL_OP_NO_SSLv2 | SSL.SSL_OP_NO_SSLv3 | SSL.SSL_OP_NO_TLSv1 |
```

Comment:
Initially, I did what you suggested, but then noticed I needed the temp variable for the call to ```setOptions``` so thought I might as well use it for ```clearOptions``` as well.  I'm not bothered either way, let me know your thoughts and I'll make the change if you still think I should.